app/routes/groups.tsx

```diff
@@ -1,26 +1,45 @@
-import { Card } from '~/components/ui/containers';
 import { useGroups } from '~/hooks/useGroups';
 import { Link } from '@remix-run/react';
-import { LinkButton } from '~/components/ui/forms';
+import { Button } from '~/components/ui/button';
+import { Image } from '~/components/ui/images';
 
 export default function Group() {
   const groups = useGroups();
   return (
-    <>
-      <Card>
-        <ul className="max-w-md space-y-1 text-gray-500 list-disc list-inside dark:text-gray-400">
+    <div className="bg-primary">
+      <div className="m-auto max-w-screen-xl">
+        <ul className="w-full flex flex-wrap justify-center text-secondary">
           {groups?.map((group) => {
             return (
-              <li key={group.id}>
-                <Link to={group.id}>{group.name}</Link>
+              <li className="flex flex-wrap justify-center my-2 mx-auto md:m-2" key={group.id}>
+                <Link to={group.id}>
+                  <div className="border-slate-100 border-2 rounded-2xl  m-1 px-4 py-4 flex flex-col justify-center items-center md:w-80 hover:transform hover:scale-105 transition-transform duration-300">
+                    <div className="w-[13.13rem] h-[13.13rem]">
+                      <Image
+                        className="bg-contain bg-center flex self-start rounded-full"
+                        src="https://images.pexels.com/photos/3184339/pexels-photo-3184339.jpeg"
+                        alt="Group meeting around an office table"
+                        background="https://res.cloudinary.com/dxctpvd8v/image/upload/v1709096811/SocialPlanit-Loading_qedebk.png"
+                        width={210}
+                        height={210}
+                      />
+                    </div>
+                    <div className="mx-2 my-4">
+                      <h2 className="font-extrabold md:text-xl my-1">{group.name}</h2>
+                      <h3>Group-headline-here</h3>
+                    </div>
+                  </div>
+                </Link>
               </li>
             );
           })}
         </ul>
-      </Card>
-      <div className="px-6 py-6 sm:px-16 max-w-sm">
-        <LinkButton to="/groups/new">Create New Group</LinkButton>
       </div>
-    </>
+      <div className="m-auto max-w-screen-xl flex justify-center items-center py-4">
+        <Button variant="warm" buttonStyle="rounded">
```

Comment:
I've never seen this pattern before (Link inside Button). Should we leave just Link?

Reply:
I was intending on using the styling of the Button component as I prefer it over the styling of the LinkButton component. 
I will use the LinkButton component in the meantime although not preferred.